### ISSUE TITLE
Failed to delete temp folder!

### ISSUE THREAD

**Issue by nsbarr** [websiddu/sync-to-slides#22]
Sync works once, but trying to sync again the upload fails and I get the above error message.

**Comment by slajax** [comment]
@nsbarr any work around here? I get this error and nothing happens.

**Comment by nsbarr** [comment]
Yes, some combo of quitting Sketch and Google Slides worker for me
